### PR TITLE
Adds Taraxa custom domain and skin

### PR DESCRIPTION
**Pull request by VargaElod23**
#### Hi! What is your PR about?

- [x] Add or edit a skin
- [x] Add a custom domain for your space
- [ ] Add an alias to migrate your space

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/taraxa-project.eth

**Comment by VargaElod23** [comment]
Hi @gokaiorg . Yeah, we're still finalizing the economical details of the strat, so I'll leave a tag here once the strategy and space is finished. Thanks for taking a look here!

**Comment by VargaElod23** [comment]
Hi @gokaiorg . We registered a dummy space at https://snapshot.org/#/taraxa-project.eth. I also made sure that all the other steps are there. Here are the PRs:

- Network addition: https://github.com/snapshot-labs/snapshot.js/pull/847
- This one for skin and domain
- Strategy: https://github.com/snapshot-labs/snapshot-strategies/pull/1190

Strategy tests need the network addition. Can you take a look? Thanks

File: skins/index.js [hunk 1]
```diff
@@ -240,6 +240,7 @@ import stakewise from "./stakewise.scss";
 import status from "./status.scss";
 import steakbank from "./steakbank.scss";
 import streamr from "./streamr.scss";
+import taraxa from "./taraxa.scss";
```
Comment (gokaiorg) [inline skins/index.js:243]: You will also need to add your skin in the `export default` index.js line 303

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by gokaiorg — APPROVED** [review]

**Comment by manny861** [comment]
0x3b484b82567a09e2588A13D54D032153f0c0aEe0

**Pull request merged**